nce_cost/README_en.md

```diff
@@ -0,0 +1,161 @@
+Running the following program need use PaddlePaddle v0.10.0 version。If your PaddlePaddle installed lower demand，Please following the [install  document ](http://www.paddlepaddle.org/docs/develop/documentation/zh/build_and_install/pip_install_cn.html) instruction upgrade PaddlePaddle install version。
```

Comment:
Running sample code in this directory requires PaddelPaddle v0.10.0 and later. If the PaddlePaddle on your device is lower than this version, please follow the instructions in [installation document](http://www.paddlepaddle.org/docs/develop/documentation/zh/build_and_install/pip_install_cn.html) and make an update.